src/main/kotlin/no/nav/familie/ef/iverksett/lagreIverksett/infrastruktur/LagreIverksettJdbc.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.familie.ef.iverksett.lagreIverksett.infrastruktur
+
+import no.nav.familie.ef.iverksett.domene.Brev
+import no.nav.familie.ef.iverksett.lagreIverksett.tjeneste.LagreIverksett
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+open class LagreIverksettJdbc(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) : LagreIverksett {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    @Transactional
+    override fun lagre(behandlingsId: UUID, iverksettJson: String, brev: List<Brev>) {
+        try {
+            return lagreIverksett(behandlingsId, iverksettJson, brev)
+        } catch (exception: Exception) {
+            secureLogger.error("Kunne ikke lagre iverksett json for behandlingsId ${behandlingsId}: ${exception}")
+            throw Exception("Feil ved LagreIverksett til basen")
+        }
+    }
+
+    private fun lagreIverksett(behandlingsId: UUID, json: String, brev: List<Brev>) {
+        val sql = "insert into iverksett values(:behandlingsId, to_json(:json::json))"
```

Comment:
Hva er egentlig forskjellen på å bruke `to_json(:json::json)` her kontra bare `:json` ? 
Jeg antar at `json: String` allerede er en gyldig json-verdi?

Reply:
Godt spørsmål, jeg vet ikke helt. Skal sjekke ut dette. Merger for nå